docs/catalog.json

```diff
@@ -1 +1 @@
omment": null}, "UNSTRUCT_EVENT_COM_SEGMENT_SCREEN_1": {"type": "OBJECT", "index": 155, "name": "UNSTRUCT_EVENT_COM_SEGMENT_SCREEN_1", "comment": null}, "CONTEXTS_COM_SNOWPLOWANALYTICS_IGLU_ANYTHING_A_1": {"type": "ARRAY", "index": 156, "name": "CONTEXTS_COM_SNOWPLOWANALYTICS_IGLU_ANYTHING_A_1", "comment": null}, "UNSTRUCT_EVENT_COM_APPLE_NOTIFICATION_EVENT_1": {"type": "OBJECT", "index": 157, "name": "UNSTRUCT_EVENT_COM_APPLE_NOTIFICATION_EVENT_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_MOBILE_APPLICATION_INSTALL_1": {"type": "OBJECT", "index": 158, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_MOBILE_APPLICATION_INSTALL_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_APPLICATION_BACKGROUND_1": {"type": "OBJECT", "index": 159, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_APPLICATION_BACKGROUND_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_APPLICATION_FOREGROUND_1": {"type": "OBJECT", "index": 160, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_APPLICATION_FOREGROUND_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_PLG_INTERACTION_1": {"type": "OBJECT", "index": 161, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_PLG_INTERACTION_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEBSITE_DEMO_REQUEST_1": {"type": "OBJECT", "index": 162, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEBSITE_DEMO_REQUEST_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_TECHOPS_TERRAFORM_ACTION_1": {"type": "OBJECT", "index": 163, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_TECHOPS_TERRAFORM_ACTION_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_ADD_TO_CART_1": {"type": "OBJECT", "index": 164, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_SNOWPLOW_ADD_TO_CART_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_CONSOLE_STRUCTURES_UI_SCHEMA_1": {"type": "OBJECT", "index": 165, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_CONSOLE_STRUCTURES_UI_SCHEMA_1", "comment": null}, "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_MONITOR_STORAGE_WAREHOUSE_GROUP_CONTEXT_1": {"type": "OBJECT", "index": 166, "name": "UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_MONITOR_STORAGE_WAREHOUSE_GROUP_CONTEXT_1", "comment": null}}, "stats": {"last_modified": {"id": "last_modified", "label": "Last Modified", "value": "2021-11-26 10:30UTC", "include": true, "description": "The timestamp for last update/change"}, "row_count": {"id": "row_count", "label": "Row Count", "value": 3051551.0, "include": true, "description": "An approximate count of rows in this table"}, "bytes": {"id": "bytes", "label": "Approximate Size", "value": 996333056.0, "include": true, "description": "Approximate size of the table as reported by Snowflake"}, "has_stats": {"id": "has_stats", "label": "Has Stats?", "value": true, "include": false, "description": "Indicates whether there are statistics for this table"}}, "unique_id": "source.snowplow_mobile.atomic.events"}}, "errors": null}
```

Comment:
FYI - It  maybe slightly cleaner to leave refreshing the docsite until the end.

Reply:
Will do for next time, I'll re-run the docs in the push to release commit on the release branch